plugin.video.rivedila7/.gitignore

```diff
@@ -0,0 +1,123 @@
+# Byte-compiled / optimized / DLL files
```

Comment:
drop from the submission

Reply:
you mean that we need remove completely the.gitignore file ? or only this first line ?

Reply:
All of it

Reply:
done